README.md

```diff
@@ -406,13 +406,14 @@ description: A cool pipeline
 author: me
 prefix: cool-pipes-company
 skip:
+  - github
   - ci
   - github_badges
```

Comment:
should we add a bit more intelligence to the code, that the template-readme checks for `github` and `github_badges` before adding them or can we assume that people would always select both anyway and this would not be a problem?

Reply:
So far, using only `github` will also remove `github_badges` and `ci`, but not the other way around. Maybe adding a small sentence explaining that will be good.